app/src/components/Map.jsx

```diff
@@ -136,13 +136,18 @@ class Map extends Component {
       return;
     }
     this.state.trackLayer.recalculatePosition();
-
     this.state.trackLayer.drawTile(
       workProps.vesselTrack.seriesGroupData,
       workProps.vesselTrack.selectedSeries,
       workProps.filters,
-      workProps.map.vesselTrackDisplayMode
+      workProps.map.vesselTrackDisplayMode,
+      // I don't understand why this array has way more points than it should
+      // workProps.vesselTrack.seriesGroupData.weight,
+      1,
+      this.state.zoom
     );
+    // this shouldn't be called when no apparent track shows up, which I don't know how to test
+    this.state.overlay.dim();
```

Comment:
We want the canvas layer to dim when a user shows a track. Problem is, for some reason `workProps.vesselTrack.seriesGroupData` / `workProps.vesselTrack.selectedSeries` always has something even when user clicked in the middle of the ocean where there are no vessels at all, so the canvas layer will dim whenever she clicks on the map.
Other (UI) thing: can we deselect a track? Right now once you select a track it's dimmed forever.